target/linux/ath79/dts/qca9563_zte_mf286ar.dtsi

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2021 Cezary Jackiewicz
+// Copyright (c) 2021, 2022 Lech Perczak
+#include "qca9563_zte_mf286.dtsi"
+
+/ {
+	ubi-concat {
+		compatible = "mtd-concat";
+		devices = <&ubiconcat0 &ubiconcat1>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				reg = <0x0 0x7a60000>;
+				label = "ubi";
+			};
+		};
+	};
+};
+
+&led_debug {
+	/* Hidden blue SMD LED below signal strength LEDs
+	 * Visible through slits underside of the case,
+	 * and slightly through the case below signal state LEDs */
+	color = <LED_COLOR_ID_BLUE>;
+	label = "blue:debug";
+};
+
+&spi {
+	flash@0 {
+		partitions {
+			partition@0 {
+				label = "u-boot";
+				reg = <0x000000 0x0a0000>;
+				read-only;
+			};
+
+			partition@a0000 {
+				label = "u-boot-env";
+				reg = <0x0a0000 0x020000>;
+				read-only;
+			};
+
+			partition@c0000 {
+				label = "reserved1";
+				reg = <0x0c0000 0x140000>;
+				read-only;
+			};
+		};
+	};
+
+	flash@1 {
+		partitions {
+			partition@0 {
+				label = "fota-flag";
+				reg = <0x00000 0xa0000>;
+				read-only;
+			};
+
+			art: partition@a0000 {
```

Comment:
Are these `art:` label (and `mac:`) used at a different place?

Reply:
They were leftovers indeed. Removed.